src/graphql/old-schema.graphql

```diff
@@ -253,6 +261,13 @@ enum AccountStatus {
 }
 
 type Mutation {
+  requestPhoneCodeGeetest(
+    phone: String @pattern(regexp: "^\\+[0-9]+$", policy: THROW)
+    geetestChallenge: String @stringLength(min: 10, max: 1000, policy: THROW) # TODO
+    geetestValidate: String @stringLength(min: 10, max: 1000, policy: THROW)
+    geetestSeccode: String @stringLength(min: 10, max: 1000, policy: THROW)
+  ): Success
```

Comment:
@jrbuhl93 Do you know where the lengths of these is documented? I haven't found it...

Reply:
I haven't found it specified either. Do you think this validation is necessary?

Reply:
Yeah we should have some limits, and in this case keep them in a big range just to be safe until we learn their real lengths.